release notes/v0.54.0.md

```diff
@@ -0,0 +1,304 @@
+k6 `v0.54.0` is here 🎉! This release includes:
+
+- A new experimental CSV module
+- new ECMAScript features
+- Updated logo and branding
+
+## Breaking changes
+
+- [#3913](https://github.com/grafana/k6/pull/3913) changes mapping of the Golang's `math/big.Int` type to `bigint` type in k6.
+- [#3922](https://github.com/grafana/k6/pull/3922) removes `lib.Min` and `lib.Max` from k6's Go API, which could affect custom extensions that rely on these functions.
```

Comment:
Do these two items require any docs updates? 🤔

Reply:
These luckily no since they are part of the k6's Golang's API (https://pkg.go.dev/go.k6.io/k6@v0.53.0/lib#Max)